File: solutions/uri/1008/1008.js
const input = require('fs').readFileSync('/dev/stdin', 'utf8')

const [_a, _b, _c] = input
  .trim()
  .split('\n')
const [a, b] = [_a, _b].map(x => parseInt(x))
const c = parseFloat(_c)

console.log(`NUMBER = ${parseInt(a)}`);
console.log(`SALARY = U$ ${(b * c).toFixed(2)}`);

File: solutions/uri/1008/1008.r
input <- file('stdin', 'r')

a <- as.integer(readLines(input, n=1))
b <- as.integer(readLines(input, n=1))
c <- as.double(readLines(input, n=1))

write(sprintf("NUMBER = %d", a), "")
write(sprintf("SALARY = U$ %.2f", b * c), "")
